### ISSUE TITLE
Return value of postProcess() is ignored on macro output

### ISSUE THREAD

**Issue by selquest** [RPTools/maptool#1699]
There appears to be an attempt to replace newline characters in macro output with `<br>`s, but the return value is ignored.  Given that this appears to have been the case for some time, I'm not sure the functionality is really needed.  It may make sense to remove the call rather than use the return value.

https://github.com/RPTools/maptool/blob/f2469c3ea99fa013349b699a8bf616fb0a3353ee/src/main/java/net/rptools/maptool/client/macro/MacroManager.java#L184

**Comment by Phergus** [comment]
That's not the only weird thing.  Just before and after the pointless call to `postprocess()` are the same lines of code:
```
        context.addTransform(key + " " + details);
        postprocess(details);

        context.addTransform(key + " " + details);
```
So the macro context transformation history gets the same info put into it twice.

@cwisniew   Any insight as to why the postprocess call is there but not used?

And prior to that are these lines which create variables that aren't used.
```
        boolean trustedPath =
            macroExecutionContext == null ? false : macroExecutionContext.isTrusted();
        String macroButtonName =
            macroExecutionContext == null
                ? "<chat>"
                : macroExecutionContext.getName() + "@" + macroExecutionContext.getSource();
```

**Comment by cwisniew** [comment]
Checking all the way back it has always been like this, 
My guess is it originally was assigned back to the variable but removed to test during development and when the test worked forgotten about and never removed.

This is only way I can explain the double up of adding the transform to the context (as it would have been a way to compare them).


The trustedPath, someone has at some stage made a code change that screwed that up. The basic idea is if every macro in the macro chain is trusted then the output can be highlighted (there are settings in preferences for this). I can't remember when it stopped working as I never used it, and the people who requested it never pointed out it stopped working.

**Comment by Phergus** [comment]
This is like code archaeology.  Look at ancient artifacts and wondering what their purpose was.  lol

**Comment by JamzTheMan** [comment]
On the opposite front I'd love to see no longer breaks on empty content...

**Comment by selquest** [comment]
> That's not the only weird thing. Just before and after the pointless call to `postprocess()` ...

Yeah, that's why I got to looking into this section of the code in the first place - I wasn't quite following a lot of what was going on here, but I was 100% confident that a pointless string manipulation that wasn't even being stored was a sign that something probably wasn't right :)
